azurerm/helpers/azure/identity.go

```diff
@@ -0,0 +1,34 @@
+package azure
```

Comment:
Could we put this id parsing of a MSI into the `azurerm/internal/services/msi/parse` directory? I suppose this functionality should belong to the `msi` service, and there are quite a few resources (those support the user assigned identities) are waiting for this parsing function.

Reply:
+1, we want to eventually move everything into the interal folder